frontend/public/components/secrets/create-secret.tsx

```diff
@@ -1485,7 +1490,10 @@ type WebHookSecretFormState = {
 
 type WebHookSecretFormProps = {
   onChange: Function;
+  onError: Function;
+  onFormDisable: Function;
+  secretType: string;
   stringData: {
-    WebHookSecretKey: string;
```

Comment:
> Not sure why this was changed? Since the Webhook secret type needs to contain this key.

@jhadvig The type error I am getting with ` WebHookSecretKey: string;` is
`Type '{ [key: string]: string; }' is not assignable to type '{ WebHookSecretKey: string; } & { [key: string]: string; }'.`

Not sure why this works in the existing code. The `stringData` [source](https://github.com/openshift/console/blob/master/frontend/public/components/secrets/create-secret.tsx#L262) property is of type index signature `'{ [key: string]: string; }'` which would accept ` WebHookSecretKey: string;` too. I tested the WebHook secret form and it look good to me. 

Leaving a note for QA team to verify the secret form thoroughly for any breaking due to few types changes in this PR. 
cc: @yapei